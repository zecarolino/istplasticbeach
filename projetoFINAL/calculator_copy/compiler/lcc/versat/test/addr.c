int var;
int main() {
	int *addr = &var;
	/* int value = 0x163;
	addr = (int*)value; */
	*addr = 12;
	return var;
}
