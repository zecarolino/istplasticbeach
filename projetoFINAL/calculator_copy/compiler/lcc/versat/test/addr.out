VCD info: dumpfile xtop.vcd opened for output.
<
Catched Trap at time  427 clock cycles:
